### PR TITLE
About page

### PR DESCRIPTION
**Pull request by RuslanHrybchuk**

### CODE REVIEW

File: src/app/component/about-page/about-page.component.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+  <div class="full-text-block">
+    <h2>
+      {{ "about-us.intro.block-1.header" | translate }}
+    </h2>
+
+    <span>
+      {{ 'about-us.intro.block-1.text' | translate }}
+    </span>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    <button>
+      {{ 'about-us.intro.block-1.button' | translate }}
+    </button>
+  </div>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+</div>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+<div class="vector-bg-2"></div>
+<div class="cup-img"></div>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+<div class="container-2">
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+  <div class="full-text-block">
+    <h2>
+      {{ 'about-us.intro.block-2.header' | translate }}
+    </h2>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+    <span>
+      {{ 'about-us.intro.block-2.text' | translate }}
+    </span>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+    <button>
+      {{ 'about-us.intro.block-2.button' | translate }}
+    </button>
+  </div>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+</div>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+<div class="steps-main-block">
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+  <div class="dashed-line">
+    <div class="leaves"></div>
+  </div>
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 2

File: src/app/component/about-page/about-page.component.html [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+  <h2>
+    {{ 'about-us.steps.title' | translate }}
+  </h2>
+
+  <div class="card-holder-odd first-card-holder">
+
+    <span class="number-title num1"></span>
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: class shouldn`t contain name with numbers

File: src/app/component/about-page/about-page.component.html [hunk 13]
```diff
@@ -0,0 +1,162 @@
...
+
+    <h3 class="card-h3">
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: change class name, not logic to add h3 in h3 tag

File: src/app/component/about-page/about-page.component.html [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+      {{ 'about-us.steps.block-1.header' | translate }}
+    </h3>
+
+    <div class="bg-image bg1"></div>
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 56

File: src/app/component/about-page/about-page.component.html [hunk 15]
```diff
@@ -0,0 +1,162 @@
...
+
+    <a class="card-a" href="">
+      {{ 'about-us.steps.block-1.button' | translate }}
+    </a>
+
+  </div>
+
+  <div class="card-holder-even">
+
+    <span class="number-title num2"></span>
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 56

File: src/app/component/about-page/about-page.component.html [hunk 16]
```diff
@@ -0,0 +1,162 @@
...
+
+    <h3 class="card-h3">
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: same as line 58

File: src/app/component/about-page/about-page.component.html [hunk 17]
```diff
@@ -0,0 +1,162 @@
...
+  </div>
+
+
+</div>
+
+<footer>
+  <app-footer></app-footer>
+  <div class="copyright">
+    &copy; Copyright {{ actualYear }}. Green City.
+  </div>
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.html]: generally delete all unneeded enters as line 2;
change class, id and picture name without numbers and tag names

File: src/app/component/about-page/about-page.component.spec.ts [hunk 18]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.spec.ts]: delete enter

File: src/app/component/about-page/about-page.component.ts [hunk 19]
```diff
@@ -0,0 +1,21 @@
...
+  public actualYear = new Date().getFullYear();
+
+  constructor(private titleService: Title) { }
+
+  ngOnInit() {
+    this.titleService.setTitle('About');
+  }
+}
+
+
```
Comment (KimAT25) [inline src/app/component/about-page/about-page.component.ts]: please fix formatting with enters

File: src/assets/i18n/uk.json [hunk 20]
```diff
@@ -494,7 +494,7 @@
         "button": "Знайти однодумців"
       },
       "block-4": {
-        "header": "Мої дії всеодно нічого\nне змінять",
+        "header": "Мої дії все одно нічого\nне змінять",
         "text": "Ви надихнетесь історіями інших  людей, які здобули екологічні звички та змінили своє життя",
         "button": "Надихнутись Історіями"
       },
```
Comment (KimAT25) [inline src/assets/i18n/uk.json]: please check align

File: src/app/component/about-page/about-page.component.spec.ts (hunk 18 above)
Comment (nabermi2020) [reply src/app/component/about-page/about-page.component.spec.ts]: not mandatory it's autogenerated

File: src/app/component/about-page/about-page.component.ts (hunk 19 above)
Comment (nabermi2020) [inline src/app/component/about-page/about-page.component.ts]: Same as line 11

File: src/app/component/about-page/about-page.component.html [hunk 21]
```diff
@@ -0,0 +1,162 @@
+<div class="container">
```
Comment (nabermi2020) [inline src/app/component/about-page/about-page.component.html]: general: empty lines can be used in some places for better separation of the code structure

Comment (nabermi2020) [reply src/app/component/about-page/about-page.component.html]: general: replace magic numbers from class names and IDs. Rename appropriate names in stylesheet

Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: Fixed

File: src/app/component/about-page/about-page.component.ts [hunk 22]
```diff
@@ -0,0 +1,21 @@
...
+import { Title } from '@angular/platform-browser';
+
+@Component({
+  selector: 'app-about-page',
+  templateUrl: './about-page.component.html',
+  styleUrls: ['./about-page.component.css']
+})
+
+export class AboutPageComponent implements OnInit {
+
```
[the remark this answers is not in the document]
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.ts]: fixed

File: src/app/component/about-page/about-page.component.ts (hunk 19 above)
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.ts]: done

File: src/app/component/about-page/about-page.component.html (hunk 17 above)
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: done

File: src/app/component/about-page/about-page.component.html (hunk 16 above)
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: done

File: src/app/component/about-page/about-page.component.html (hunk 2 above)
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: done

File: src/app/component/about-page/about-page.component.html (hunk 1 above)
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: done

File: src/app/component/about-page/about-page.component.html [hunk 23]
```diff
@@ -0,0 +1,162 @@
...
+
+  <div class="illustration-man"></div>
+  <div class="leaf"></div>
+  <div class="vector-bg"></div>
+
+  <div class="full-text-block">
+    <h2>
+      {{ "about-us.intro.block-1.header" | translate }}
+    </h2>
+
```
[the remark this answers is not in the document]
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: done

File: src/app/component/about-page/about-page.component.html [hunk 24]
```diff
@@ -0,0 +1,162 @@
+<div class="container">
+
+  <div class="illustration-man"></div>
+  <div class="leaf"></div>
+  <div class="vector-bg"></div>
+
```
[the remark this answers is not in the document]
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: done

File: src/app/component/about-page/about-page.component.html [hunk 25]
```diff
@@ -0,0 +1,162 @@
+<div class="container">
+
```
[the remark this answers is not in the document]
Comment (RuslanHrybchuk) [reply src/app/component/about-page/about-page.component.html]: done

**Pull request merged**